Mathlib/Algebra/Module/LocalizedModule/Basic.lean

```diff
@@ -481,8 +441,8 @@ variable (S M)
 @[simps]
 noncomputable def mkLinearMap : M →ₗ[R] LocalizedModule S M where
   toFun m := mk m 1
-  map_add' x y := by simp [mk_add_mk]
-  map_smul' _ _ := (smul'_mk _ _ _).symm
+  map_add' x y := by simp
+  map_smul' r m := by simp [mk, OreLocalization.smul_oreDiv]
```

Comment:
Why specify the arguments?

Reply:
No reason in particular. I have reverted this change.